docs/content/getting-started.md

```diff
@@ -25,12 +25,8 @@ This guide will show you
 
 ### Downloading Paket's Bootstrapper
 
-1. Create a `.paket` directory in the root of your solution.
-1. Download the latest
-   [`paket.bootstrapper.exe`](https://github.com/fsprojects/Paket/releases/latest)
-   into that directory.
-1. Rename `.paket/paket.bootstrapper.exe` to `.paket/paket.exe`.
-   [Read more about "magic mode"](bootstrapper.html#Magic-mode).
+1. In the root directory of your solution, execute `curl -fsSL https://raw.githubusercontent.com/fsprojects/Paket/master/install/web-install.fsx | fsharpi --quiet --readline-`
```

Comment:
This URL of course assumes this PR is merged and the script exists, so if you want to test it out, use this instead:

    curl -fsSL https://raw.githubusercontent.com/jwosty/Paket/web-install/install/web-install.fsx | fsharpi --quiet --readline-

Reply:
![image](https://user-images.githubusercontent.com/57396/38718599-13873d34-3eee-11e8-8ef8-1da0c039441b.png)

^^ that's why we never did that

Reply:
Would a PowerShell script suffice for Windows?